x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/AllSupportedFieldsTestCase.java

```diff
@@ -923,7 +951,7 @@ private static boolean supportedInEnrich(DataType t) {
         return switch (t) {
             // Enrich policies don't work with types that have mandatory fields in the mapping.
             // https://github.com/elastic/elasticsearch/issues/127350
-            case AGGREGATE_METRIC_DOUBLE, SCALED_FLOAT,
+            case AGGREGATE_METRIC_DOUBLE, SCALED_FLOAT, EXPONENTIAL_HISTOGRAM,
```

Comment:
Is there a separate issue for ENRICH support for exponential_histogram that we could/should link here, or does https://github.com/elastic/elasticsearch/issues/127350 extend to exponential_histogram?